reference/path-parameters/merchant_account_id.yaml

```
@@ -0,0 +1,9 @@
+---
+schema:
+  type: string
+  format: uuid
+  example: 8724fd24-5489-4a5d-90fd-0604df7d3b83
+name: id
+in: path
+required: true
+description: The unique ID for a merchant account.
```

```suggestion
schema:
  type: string
  example: plantly_uk
name: id
in: path
required: true
description: The unique ID for a merchant account.
```

We did not want to use UUIDs for merchant accounts.